### DISCUSSION TITLE
Search display not correct until screen is refreshed

### DISCUSSION THREAD

**Discussion by WoodieFL** [code4recovery/12-step-meeting-list#842] [Q&A] [answered]
When viewing meetings on our website, if any search criteria is selected, i.e., Region, Day, Time, Type, the Address and Region columns are incorrect. Region is displaying in the address column and nothing is in the Region column. When I refresh the screen, everything is correct. 

Attached are screenshots before and after refreshing the screen.

![Search by day before refresh](https://user-images.githubusercontent.com/73310072/175574864-972be93e-e070-4dc0-b263-ff01d6049eea.png)

![Search by day after refresh](https://user-images.githubusercontent.com/73310072/175575492-e0207b65-9ede-47b2-8660-30573720062f.png)

**Comment by joshreisner** [comment, marked as answer]
sorry about that @WoodieFL ! it will be fixed in the next plugin release, coming soon 🙏 

also discussed here: https://github.com/code4recovery/12-step-meeting-list/discussions/835

**Reply by WoodieFL** [reply]
Sorry for the edits, but I kept seeing things. Thanks for the quick reply.
